platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/transaction/TransactionWrapper.java

```diff
@@ -117,6 +133,12 @@ public EventTransaction getTransaction() {
         return payload;
     }
 
+    @Override
+    @NonNull
+    public Bytes getTransactionsBytes() {
+        return payloadBytes;
+    }
+
```

Comment:
during migration, some events will have old transactions, some will have new, so I'm not sure if we should force the app to check every time, so maybe we should have a method that returns one or the other, depending on which one is available

Reply:
To mitigate this we can continue using `getTransaction()` that returns EventTransaction, until we have completely switched to the Bytes format. Anyway, I will adapt the method to return the bytes of the `ApplicationTransaction` if applicable, so that it covers fully all cases.